tests/manage/test_verify_all_fields_in_sc_yaml_with_oc_describe_sc.py

```diff
@@ -0,0 +1,131 @@
+import pytest
+import logging
+
+from tests import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility import templating
+from ocs_ci.framework.testlib import ManageTest, tier1
+from tests.fixtures import create_ceph_block_pool
+
+log = logging.getLogger(__name__)
+RBD_SC_OBJ = None
+CEPHFS_SC_OBJ = None
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    This is a test fixture
+    """
+    def finalizer():
+        teardown()
+    request.addfinalizer(finalizer)
+    setup()
+
+
+def setup():
+    """
+    Setting up the environment
+    *. Creating RBD and CephFS Secret
+    *. Creating Ceph block pool
+    """
+    global RBD_SECRET_OBJ
+    log.info("Creating RBD Secret")
+    RBD_SECRET_OBJ = helpers.create_secret(constants.CEPHBLOCKPOOL)
+
+    log.info("Creating CEPHFS Secret")
+    global CEPHFS_SECRET_OBJ
+    CEPHFS_SECRET_OBJ = helpers.create_secret(constants.CEPHFILESYSTEM)
```

Comment:
Why not use `create_rbd_secret` fixture from `contest.py` and create there also fixture `create_cephfs_secret`?

Reply:
Done